vaadin-grid-pro-flow-parent/vaadin-grid-pro-flow-integration-tests/src/main/java/com/vaadin/flow/component/gridpro/tests/MainView.java

```diff
@@ -35,6 +36,13 @@ protected void createEditorColumns() {
         Div eventsPanel = new Div();
         eventsPanel.setId("events-panel");
 
+        Div selectionPanel = new Div();
+        selectionPanel.setId("selection-panel");
+
+        UI.getCurrent().getElement().executeJs(
+                "document.addEventListener('selectionchange', () => $0.textContent = document.getSelection().toString())",
+                selectionPanel.getElement());
```

Comment:
Tried to check the selection from `document.getSelection().toString()` on the IT with the `executeScript` method, but it was always returning `null`, so I came up with this solution instead.

Reply:
Did you maybe forget to add a return?

The following IT works fine for me:
```java
    @Test
    public void gridWithCustomEditors_navigateToWithTabKey_textIsSelected() {
        var cell_0_4 = grid.getCell(0, 4);
        assertCellEnterEditModeOnDoubleClick(0, 4, "vaadin-combo-box");

        waitUntil(e -> cell_0_4.$("vaadin-combo-box").exists());
        var input = cell_0_4.$("vaadin-combo-box").first();
        input.sendKeys(Keys.TAB);

        String selection = (String) getCommandExecutor().executeScript("return document.getSelection().toString()");
        Assert.assertEquals("person1@vaadin.com", selection);

        var cell_0_5 = grid.getCell(0, 5);
        input = cell_0_5.$("input").first();
        input.sendKeys(Keys.TAB);

        selection = (String) getCommandExecutor().executeScript("return document.getSelection().toString()");
        Assert.assertEquals("2019", selection);
    }
```

Reply:
Oh, yeah. Forgot to add `return` on the JS expression. Now it works, thanks!